org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/preferences/OptionsConfigurationBlock.java

```diff
@@ -528,7 +529,7 @@ protected Button addCheckBoxWithLink(Composite parent, final String label, Key k
 		final Button checkBox= new Button(composite, SWT.CHECK);
 		checkBox.setFont(JFaceResources.getDialogFont());
 		gd= new GridData(GridData.FILL, GridData.CENTER, false, false);
-		int offset= Util.isMac() ? -4 : Util.isLinux() ? 4 : /* Windows et al. */ 3;
+		int offset= Util.isMac() ? -4 : Util.isLinux() ? 4 : Util.isFreeBSD() ? 4 : /* Windows et al. */ 3;
```

Comment:
Couldn't this be just an or?
```suggestion
		int offset= Util.isMac() ? -4 : (Util.isLinux() || Util.isFreeBSD()) ? 4 : /* Windows et al. */ 3;
```